tox.ini

```diff
@@ -1,10 +0,0 @@
-[tox]
-envlist = py27,py36
-
-[testenv]
-setenv =
-    PYTHONDONTWRITEBYTECODE = 1
```

Comment:
This didn't get ported over. Do you still want it?

Reply:
Nah, it's not really an issue outside of local development.